apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -916,6 +927,13 @@ def create_subnet_precommit(self, context):
         subnet = context.current
         network = context._plugin.get_network(context._plugin_context,
                                               subnet['network_id'])
+        context.subnet_scope = self.net_cons.get_subnet_scope(
+            str(self.name_mapper.tenant(context, network['tenant_id'])),
+            network['name'],
```

Comment:
The use-case is motivated by external networks, but there is no reason to limit it to external networks.